models/user.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
+var bcrypt = require('bcrypt');
+
+var userSchema = new mongoose.Schema({
+  name: String,
+  email: String,
+  password: String
+});
+
+userSchema.set('toJSON', {
+  transform: function(doc, ret, options) {
+    delete ret.password;
+    return ret;
+  }
+});
+
```

Comment:
Evan, your auth wasn't working properly because you hadn't added the ```authenticated``` method to the user model. Add this here on line 17:

```
userSchema.methods.authenticated = function(password, callback) {
  bcrypt.compare(password, this.password, function(err, res) {
      if (err) {
        callback(err);
      } else {
        callback(null, res ? this : false);
      }
  });
}
```